### PR TITLE
Add docs for configuring trino catalogs

### PR DESCRIPTION
**Pull request by michaeljguarino**
## Summary
This is a manual step so if we can at least point to it, that could help

## Test Plan
n/a

## Checklist

- [ ]  No images hosted from dockerhub
- [ ]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [ ]  all databases should have dashboards
    - [ ]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [ ]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [ ]  Are scaling runbooks present
    - [ ]  all databases **must** have scaling runbooks
    - [ ]  you can use the charts in `pluralsh/module-library` to accelerate this
- [ ]  do you need to add config overlays?
    - [ ]  inputing secrets
    - [ ]  configuring autoscaling
- [ ]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable
- [ ]  All major clouds must be supported
    - [ ]  Azure
    - [ ]  AWS
    - [ ]  GCP

### CODE REVIEW

File: airflow/helm/airflow/values.yaml [hunk 1]
```diff
@@ -27,8 +27,8 @@ postgres:
 
 exporter:
   image:
-    repository: gcr.io/pluralsh/prom/statsd-exporter
-    tag: v0.20.2
```
Comment (DavidSpek) [inline airflow/helm/airflow/values.yaml:31]: the tag should be `v0.23.0` based on the image vendoring.

File: airflow/helm/airflow/values.yaml [hunk 2]
```diff
@@ -54,6 +54,7 @@ exporter:
     additionalLabels: {}
 
   statsd:
+    relayAddress: "datadog.datadog:8125"
```
Comment (DavidSpek) [inline airflow/helm/airflow/values.yaml:57]: Shouldn't this be optional and only set if datadog is installed?

**Review by DavidSpek — APPROVED** [review]
One small change, other than that it looks good.

File: airflow/helm/airflow/values.yaml (hunk 1 above)
Comment (michaeljguarino) [reply airflow/helm/airflow/values.yaml:31]: it's not in our registry yet I think

File: airflow/helm/airflow/values.yaml (hunk 2 above)
Comment (michaeljguarino) [reply airflow/helm/airflow/values.yaml:57]: udp is fire and forget so it should be fine to put it in

**Pull request merged**